admission_control/admission-control-service/src/lib.rs

```diff
@@ -19,7 +19,7 @@ extern crate prometheus;
 #[path = "unit_tests/admission_control_service_test.rs"]
 mod admission_control_service_test;
 
-#[cfg(feature = "fuzzing")]
```

Comment:
the admission control fuzzer's tests were not being ran previously, fixing this (tests pass)